.gitignore

```diff
@@ -3,3 +3,4 @@
 *~
 /bazel-*
 .DS_Store
+.idea/
```

Comment:
For the record, I'm okay with this because this is present in k/k's [gitignore](https://github.com/kubernetes/kubernetes/blob/983cf51c53431170e11d1d41a820ee6bd7bd4bcc/.gitignore#L13) and afaik generally needed for folks working with JetBrains IDEs.

Reply:
That's why I was adding this (and only this) line 😃